### PR TITLE
add support for Filter by ts and value

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

File: redistimeseries/client.py [hunk 1]
```diff
@@ -362,18 +372,24 @@ def deleterule(self, source_key, dest_key):
         """Deletes a compaction rule"""
         return self.redis.execute_command(self.DELETERULE_CMD, source_key, dest_key)
 
-    def __range_params(self, key, from_time, to_time, count, aggregation_type, bucket_size_msec):
+    def __range_params(self, key, from_time, to_time, count, aggregation_type, bucket_size_msec,
+                       filter_by_ts, filter_by_value):
```
Comment (danni-m) [inline redistimeseries/client.py:376]: @AvitalFineRedis I think that having `filter_by_value` as a single parameter will make it hard on users to understand the actual type/meaning. Can we split it to 2 parameters?

File: redistimeseries/client.py [hunk 2]
```diff
@@ -362,18 +372,24 @@ def deleterule(self, source_key, dest_key):
...
+            self.appendFilerByValue(params, filter_by_value[0], filter_by_value[1])
         self.appendCount(params, count)
         if aggregation_type is not None:
             self.appendAggregation(params, aggregation_type, bucket_size_msec)
+
         return params
 
     def range(self, key, from_time, to_time, count=None,
-                aggregation_type=None, bucket_size_msec=0):
+                aggregation_type=None, bucket_size_msec=0, filter_by_ts=None, filter_by_value=None):
```
Comment (danni-m) [inline redistimeseries/client.py:392]: same as a above regarding filter_by_value

File: redistimeseries/client.py [hunk 3]
```diff
@@ -421,8 +447,8 @@ def __mrange_params(self, aggregation_type, bucket_size_msec, count, filters, fr
         params += filters
         return params
 
-    def mrange(self, from_time, to_time, filters, count=None,
-                     aggregation_type=None, bucket_size_msec=0, with_labels=False):
+    def mrange(self, from_time, to_time, filters, count=None, aggregation_type=None,
+               bucket_size_msec=0, with_labels=False, filter_by_ts=None, filter_by_value=None):
```
Comment (danni-m) [inline redistimeseries/client.py:451]: same as above regarding filter_by_value

**Review by danni-m — COMMENTED** [review]
@AvitalFineRedis Looks good, i have a small comment regarding the filter_by_value API .

File: redistimeseries/client.py (hunk 1 above)
Comment (AvitalFineRedis) [reply redistimeseries/client.py:376]: sure

**Review by danni-m — APPROVED** [review]

**Comment by danni-m** [comment]
@AvitalFineRedis LGTM

**Pull request merged**